### ISSUE TITLE
allow formating of multiple files

### ISSUE THREAD

**Issue by fawaf** [hashicorp/terraform#19442]
### Current Terraform Version

```
Terraform v0.11.8
```

### Use-cases
format more than one file at a time, but not an directory. especially since `fmt` tries to format every file in the `.terraform` directory as well.

### Attempted Solutions
in order to format more than one file, one has to use a loop or specifiy `terraform fmt` on each file separately since the `fmt` command only supports format of an entire directory or a single file.

### Proposal
support `terraform fmt file [file ..]` so one can format more that one file at once, but not an entire directory.

**Comment by gmunguia** [comment]
This would make using tools like https://github.com/okonet/lint-staged much easier.

**Comment by eryon** [comment]
Looks like we have a potential fix in #19528. Can this be looked at by a contributor? @teamterraform

**Comment by dphang** [comment]
Just came across this and I am also using `lint-staged`. If anyone is looking for a workaround while the PR is merged, and you have `node` available (assuming you probably do if you are using `lint-staged`), I just created a simple script to wrap `terraform fmt` in a loop:

Create a script, for example, I called it `terraform-multi-fmt.js`:

```js
const execSync = require("child_process").execSync;
const paths = process.argv.slice(2); // the command run is node terraform-multi-fmt.js <path1> <path2> ..., so slice(2) creates an array of ["path1", "path2", ...]

for (const path of paths) {
  execSync(`terraform fmt ${path}`, { stdio: "inherit" });
}
```

Then you can add to `lint-staged` in `package.json`:

```json
  "lint-staged": {
    "**/*.tf": [
      "node terraform-multi-fmt.js"
    ]
  }
```

**Comment by 1mamute** [comment]
Up, I'm using lint-staged too and needed to use @dphang workaround. The https://github.com/hashicorp/terraform/pull/19528 could solve the problem but has been neglected for a long time.

**Comment by RichardSilveira** [comment]
I have updated the glob pattern to ignore the .terraform directory, and it works. :partying_face: 

```json
"lint-staged": {
    "!(.terraform/**)*.tf": [
      "terraform fmt -recursive -diff"
    ]
  }
 ```

**Comment by sbevels** [comment]
This appears to be resolved in v1.4.4.
